plugins/core/build.gradle.kts

```
@@ -38,6 +38,7 @@ dependencies {
         because("It has ready to use HttpException class")
     }
     implementation(Square.moshi.kotlinReflect)
+    implementation("com.google.cloud:google-cloud-storage:1.113.2")
```

```suggestion
    implementation("com.google.cloud:google-cloud-storage:_")
```
I think you expected that first feedback!